modules/tls-profiles-view-details.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * security/tls-security-profiles.adoc
+
+[id="tls-profiles-view-details_{context}"]
+= Viewing TLS security profile details
+
+You can view the minimum TLS version and ciphers for the predefined TLS security profiles for each of the following components: Ingress Controller, Kubernetes control plane, and kubelet.
+
+[IMPORTANT]
+====
+The minimum TLS version and list of ciphers for a profile might differ between components.
```

Comment:
suggestion: I'd call it "the effective configuration of minimum TLS version and list of ciphers for a profile"